.travis.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Xaptum, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License
+
+sudo: required
+
+services:
+  - docker
+
```

Comment:
Ha, I walk back my previous comment: we probably want to have `language: cpp`, just so that the builds on the travis website don't say "Ruby" (I guess that's their default...)

Reply:
I saw that and thought that looked a little funny too. Should I specify compiler for each build that way it shows up in Travis?

Reply:
I was debating that, too. I think no, because we're already showing that via the environment variable (which shows up in the build UI), and it could get inconsistent (e.g. if someone changes the travis setting and thinks that actually changes the compiler)